sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -102,6 +102,13 @@ public partial class ManagedIdentityCredential : Azure.Core.TokenCredential
         public override Azure.Core.AccessToken GetToken(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Core.AccessToken> GetTokenAsync(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class SdkAuthFileCredential : Azure.Core.TokenCredential
```

Comment:
Do you think the type name should be simplified to just `AuthFileCredential` prepending Sdk seems redundant as every class is in the SDK

Reply:
Works for me.